cellbender/remove_background/data/dataset.py

```diff
@@ -848,11 +858,15 @@ def get_matrix_from_cellranger_mtx(filedir: str) \
         matrix_file = os.path.join(filedir, 'matrix.mtx')
         gene_file = os.path.join(filedir, "genes.tsv")
         barcode_file = os.path.join(filedir, "barcodes.tsv")
+        logging.info(f"matrix file path is {matrix_file}")
+        logging.info(f"gene file path is {gene_file}")
+        logging.info(f"barcode file path is {barcode_file}")
```

Comment:
Just to keep the scope of this PR nice and targeted, I might suggest removing changes to `get_matrix_from_cellranger_mtx()`

Reply:
Whoops, did not mean to leave those logger instances. I added them for debugging at some point. I'll cut them out. Thanks!